aspnetcore/fundamentals/logging/index.md

```diff
@@ -753,6 +753,7 @@ Third-party logging frameworks that work with ASP.NET Core:
 * [KissLog.net](https://kisslog.net/) ([GitHub repo](https://github.com/catalingavan/KissLog-net))
 * [Loggr](http://loggr.net/) ([GitHub repo](https://github.com/imobile3/Loggr.Extensions.Logging))
 * [NLog](http://nlog-project.org/) ([GitHub repo](https://github.com/NLog/NLog.Extensions.Logging))
+* [Sentry](http://sentry.io/) ([GitHub repo](https://github.com/getsentry/sentry-dotnet))
```

Comment:
Use the secure, redirected endpoint: `https://sentry.io/welcome/`

Reply:
Oh my bad. Typed in and forgot the 's'.